File: benchmarks/reference/python/fib_stress.py
"""fib_stress.py — Recursive fibonacci(40) benchmark.
Equivalent to examples/il/benchmarks/fib_stress.il"""
import sys

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

sys.exit(fib(40) & 0xFF)

File: benchmarks/reference/lua/fib_stress.lua
-- fib_stress.lua -- Recursive fibonacci(40) benchmark.
-- Equivalent to examples/il/benchmarks/fib_stress.il

local function fib(n)
    if n <= 1 then return n end
    return fib(n - 1) + fib(n - 2)
end

local result = fib(40)
os.exit(result & 0xFF)
